static/js/https/domains.js

```diff
@@ -79,118 +83,168 @@ $(document).ready(function () {
 
   // Construct a sentence explaining the HTTP situation.
   var httpDetails = function(data, type, row) {
+
     if (type == "sort")
       return null;
 
     var https = row.https.uses;
     var behavior = row.https.enforces;
     var hsts = row.https.hsts;
     var hsts_age = row.https.hsts_age;
+    var preloaded = row.https.preloaded;
+    var grade = row.https.grade;
+
+    var tls = [];
+
+    // If an SSL Labs grade exists at all...
+    if (row.https.grade >= 0) {
+
+      if (row.https.sig == "SHA1withRSA")
+        tls.push("Certificate uses a " + l("sha1", "weak SHA-1 signature"));
+
+      if (row.https.ssl3 == true)
+        tls.push("Supports the " + l("ssl3", "insecure SSLv3 protocol"));
+
+      if (row.https.tls12 == false)
+        tls.push("Lacks support for the " + l("tls12", "most recent version of TLS"));
+    }
+
+    // Though not found through SSL Labs, this is a TLS issue.
+    if (https == 1)
+      tls.push("Certificate chain not valid for all public clients. See " + l(labsUrlFor(row.canonical), "SSL Labs") + " for details.");
```

Comment:
Right now this just flags all certificate chain errors. Very soon, I'd like to start surfacing expired certificate warnings in a different way, since we have that data now in `pshtt`. It would also be ideal to distinguish incomplete chains from untrusted roots, but I'm not sure offhand how we'd do that. `pshtt` does not currently make that distinction.